armi/_bootstrap.py

```diff
@@ -34,9 +34,6 @@
     )
 
 
-from armi import context
```

Comment:
Awesome!

How did this PR allow us to remove `context` from our bootstrapping?  I love the change, but don't really see the connection.

Reply:
Added back and created an issue on this to track #1037